src/components/Animal/Animal.js

```diff
@@ -1,3 +1,11 @@
-export default function Animal() {
-  return <></>;
+import './Animal.css';
+export default function Animal(props) {
+  // return <></>;
+  return (
+    <div className="animal" style={{ top: `${props.top}`, left: `${props.left}` }}>
+      <img src={`${process.env.PUBLIC_URL}/animals/${props.type}.svg`} />
```

Comment:
You don't need the process.env.PUBLIC_URL.

animals/${props.type}.svg should work